src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
Oh, I see that's on the `queryResults`. It'd be nice if it acted on the job like I did for Python. https://github.com/GoogleCloudPlatform/google-cloud-python/pull/3844